cmd/rook/ceph/agent.go

```diff
@@ -62,7 +62,7 @@ func startAgent(cmd *cobra.Command, args []string) error {
 	agent := agent.New(context)
 	err = agent.Run()
 	if err != nil {
-		rook.TerminateFatal(fmt.Errorf("failed to run rook ceph agent. %+v\n", err))
+		rook.TerminateFatal(fmt.Errorf("failed to run rook ceph agent. %+v", err))
```

Comment:
why remove the newline? just a minor thing, but seems like it made the pod logs look better

Reply:
@travisn  go linter give errors if the error strings are ended with newline or punctuation. thats why its removed in this PR.